src/node/rpc/handlerregistry.h

```diff
@@ -92,6 +93,24 @@ namespace ccf
         return *this;
       }
 
+      // If true, caller does not need to be authenticated
```

Comment:
This sounds like exactly the same thing as `require_client_signature`. The comments around its use-point also talk about _requiring_ auth. Convince me these are different?

Reply:
The two flags are different. `require_client_signature` indicates that the RPC requires a signed command (i.e. use `scurl.sh`) while `caller_auth_disabled` indicates that the client does not need to be authenticated (i.e. not specifying `--key` and `--cert` to `curl` is OK), even on a frontend that has (through its registry) a `certs` table. In practice however, I imagine that the two are not completely orthogonal (e.g. using proxies).

Reply:
Both could be considered authentication, if we had a mechanism to convey intended identity on signed messages?